.gitignore

```diff
@@ -59,3 +59,5 @@ UpgradeLog*.htm*
 packages/
 GitExtensions.*.sln.VisualState.xml
 GitExtensions.settings.backup
+Bin/Git-1.9.5-preview20141217.exe
```

Comment:
Please consider Bin/Git-* pattern that includes future versions as well.
The same for KDiff3.